utils/Makefile

```
@@ -61,6 +61,7 @@ config.ml: config.mlp $(ROOTDIR)/Makefile.config Makefile
 	    $(call SUBST_STRING,FLEXLINK_FLAGS) \
 	    $(call SUBST_QUOTE,FLEXDLL_DIR) \
 	    $(call SUBST,HOST) \
+            $(call SUBST_STRING,BINDIR) \
```

Highly pedantically, this should be:
```suggestion
	    $(call SUBST_STRING,BINDIR) \
```
(a tab followed by four spaces)